pycaret/internal/pycaret_experiment/time_series_experiment.py

```diff
@@ -364,7 +366,7 @@ def _check_and_clean_index(
             internally converted to datetime using `pd.to_datetime()`. If None,
             then the data's index is used as is for modeling.
 
-        seasonal_period : Optional[Union[int, str]], default = None
+        seasonal_period : Optional[Union[List[Union[int, str]], int, str]], default = None
```

Comment:
Now that this is more complex, it may be good to add some examples here for the different use cases. What do you think?